components/scream/src/physics/p3/p3_main_impl.hpp

```diff
@@ -73,7 +73,7 @@ void Functions<S,D>
 ::p3_main_part1(
   const MemberType& team,
   const Int& nk,
-  const bool& log_predictNc,
+  const bool& predictNc,
```

Comment:
Out of curiosity, why did you rename log_X to X. Don't get me wrong - I hated log_X because I always thought we were taking the log of something - but it just seems orthogonal to the point of this PR and I thought @xyuan was going to switch these logicals to is_X in an upcoming PR.

Reply:
Coincidentally, I added a bullet to the description above about this. I guess I couldn't help myself since I was already dealing with these variables in the struct. I can change them back if you think it complicates the aforementioned PR. (though I think simply replacing `log_` with `is_` might produce some awkward pseudo-English [though software devs don't seem to mind this particular outcome]).

Reply:
+1 for the doubt on whether `is_predictNc` is better. Imho the "is_" prefix works if the following word is an adjective (or a noun at best), not a verb. For verbs, no prefix is better: `computeA` is better than `is_computeA`.

Reply:
Yeah, I noticed that is_predictNC sounds horrible. I'm caught between wanting to unambiguously signal that a variable is a logical and not wanting to sound like a russian troll. Maybe do_predictNC? In any case, I think your change is fine - I mostly just wanted to tag Xingqiu so he's aware of the change when he looks for these logical variables.

Reply:
I gues "do_blah" is ok. I don't have a strong opiion between options like "computeA" or "do_computeA". Whatever you guys decide there, works for me.

Reply:
To me, the verb is enough of an indicator, but I don't feel strongly enough to oppose this idea. I do kind of dislike the mixing of snake case and camel case, but that's a whole other can of worms. :-)

Reply:
Yeah, camel-snake is an abomination. Let's not do anything here and instead fight this battle in Xingqiu's PR. I'm cool with either way as well.